contrib/lang/python/packages.el

```diff
@@ -34,9 +27,8 @@ which require an initialization must be listed explicitly in the list.")
 
 (defun python/init-company-anaconda ()
   (use-package company-anaconda
-    :if (boundp 'company-backends)
-    :defer t
-    :init (add-to-list 'company-backends 'company-anaconda)))
+    :init (add-hook 'python-mode-hook 'company-mode)
```

Comment:
What is the motivation behind the trade of `:defer t` with a hook in `:init` ? `company-anaconda` has an autoload cookie.